src/test/java/delete_task_use_case/DeleteTaskControllerTest.java

```diff
@@ -0,0 +1,36 @@
+package delete_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.DataAccessTask;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DeleteTaskControllerTest {
+    private final static DeleteTaskPresenter outputBoundary = new DeleteTaskPresenter();
+    private final static DeleteTaskDataAccess dataAccess = new DeleteTaskDataAccess();
+    private final static DeleteTaskInteractor inputBoundary = new DeleteTaskInteractor(outputBoundary, dataAccess);
+    private final static DeleteTaskController controller = new DeleteTaskController(inputBoundary);
+    private final static int dayIndex = 5;
+    private final static String taskTitle = "Task";
+
+    @Test
+    void testDeleteTaskController() {
+        DataAccessTask task = new DataAccessTask("Task");
+        HashMap<String, DataAccessTask> tasks = new HashMap<>();
+        HashMap<String, DataAccessEvent> events = new HashMap<>();
+        tasks.put("Task", task);
+
+        DataAccessDay day = new DataAccessDay(tasks, events);
+        dataAccess.getDays().set(5, day);
+        DeleteTaskOutputData outputData = controller.deleteTask(dayIndex, taskTitle);
+
+        assertEquals(5, outputData.getDayIndex());
+        assertEquals("Task", outputData.getTaskTitle());
+        assertTrue(outputData.isSuccess());
+        assertTrue(dataAccess.getDays().get(5).getTasks().isEmpty());
```

Comment:
You should technically whether a Task instance by the name "Task" still exists for the Day object at day index 5.

Reply:
Added this test scenario.